app/models/solidus/affirm/checkout_payload.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_model_serializers'
```

Comment:
I expected that to be the case as well, during testing and running the sandbox I got an error on the inheritance here. Hence the require statement.